src/GitHub.Exports.Reactive/ViewModels/IRepositoryForm.cs

```diff
@@ -25,8 +25,8 @@ public interface IRepositoryForm : IViewModel
         /// dashes.
         /// </summary>
         string SafeRepositoryName { get; }
-        ReactivePropertyValidator<string> RepositoryNameValidator { get; }
-        ReactivePropertyValidator<string> SafeRepositoryNameWarningValidator { get; }
+        ReactivePropertyValidator<(string repositoryName, IConnection connection, IAccount account)> RepositoryNameValidator { get; }
```

Comment:
Looking at the code, I can't see where we're using the `connection` or `account` for validation. I wonder if once upon a time we were checking for an existing repository with the same name as part of the validation? It looks like this check is now done and the error surfaced when the user attempts to create the repository.

I think we might be able to simplify this to use just the `repositoryName`. I'm wondering if the buggy code was actually completely obsolete. ;-)